deployment/SampleClaimsProvider/Controllers/V20210519/ClaimsProviderController.cs

```diff
@@ -0,0 +1,136 @@
+﻿// ---------------------------------------------------------------------------------
+// <copyright company="Microsoft">
+//   Copyright (c) {Microsoft} Corporation. All rights reserved.
+// </copyright>
+// ---------------------------------------------------------------------------------
+
+namespace SampleClaimsProvider
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Primitives;
+    using Newtonsoft.Json;
+
+    [Route("api/claims/claimInfo")]
+    [ApiController]
+    public class ClaimsProviderController : ControllerBase
+    {
+        private readonly IClaimsProviderStore claimsProviderStore;
+        private readonly ILogger<ClaimsProviderController> logger;
+
+        public ClaimsProviderController(ILogger<ClaimsProviderController> logger, IConfiguration configuration, IClaimsProviderStore claimsProviderStore)
+        {
+            this.logger = logger;
+            this.claimsProviderStore = claimsProviderStore;
+        }
+
+        /// <summary> This API retrieves claims for a specified vehicleId.
+        /// <example>
+        /// [GET] https://{hostname}/api/claims/claimInfo/vehicles/testVehicle1?paths=path1
```

Comment:
This also needs to document how we can get all the claims for a single vehicle, not just one path.
That should be simply documenting a `paths=*` I believe.